CMakeLists.txt

```diff
@@ -69,15 +69,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 set(SPIRV_TOOLS_BINARY_DIR "$<TARGET_FILE_DIR:spirv-dis>")
 
-if (NOT MSVC)
-  set(ENABLE_CLSPV_OPT ON)
-else()
-  # clspv-opt is not supported on Windows, because LLVM opt does not support
-  # out-of-tree plugins on Windows (http://go/so/41943353).
-  # TODO(dnovillo): Need a different implementation that does not rely on
-  #                 LLVM opt.
-  set(ENABLE_CLSPV_OPT OFF)
-endif()
+option(ENABLE_CLSPV_OPT "Enable the clspv-opt driver." ON)
```

Comment:
I was trying an experiment to generate some code without optimizations, does this flag affect the `clspv` target at the moment? I built with this on and off, I get identical output for `clspv`.

Reply:
This controls whether to build the standalone executable clspv-opt. It now builds on all platforms by default. The executable just lets us run individual passes on IR (for testing purposes). You can -O0 as an input to clspv to disable optimizations, but there are no guarantees it generates correct code.